.gitignore

```diff
@@ -1,2 +1,3 @@
 coverage
 node_modules
+package-lock.json
```

Comment:
Can you ignore yarn-lock here and put `package-lock=false` in .npmrc?

Reply:
Done.